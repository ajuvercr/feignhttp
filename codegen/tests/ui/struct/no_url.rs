use feignhttp_codegen::feign;

// error: metadata url not specified
//    |  #[feign(aaa = "http://xxx")]
//    |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

struct Http;

#[feign(aaa = "http://xxx")]
impl Http {}

fn main() {}
