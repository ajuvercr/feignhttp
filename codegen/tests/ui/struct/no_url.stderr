error: metadata url not specified
 --> tests/ui/struct/no_url.rs:9:1
  |
9 | #[feign(aaa = "http://xxx")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `feign` (in Nightly builds, run with -Z macro-backtrace for more info)
